srcpkgs/python3-MechanicalSoup/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'python3-MechanicalSoup'
+pkgname=python3-MechanicalSoup
+_pkgname=MechanicalSoup
+version=0.11.0
+revision=1
+wrksrc=${_pkgname}-${version}
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+short_desc="Python library for automating interaction with websites"
+maintainer="Alex Childs <misuchiru03+void@gmail.com>"
+license="MIT"
+homepage="https://github.com/MechanicalSoup/MechanicalSoup"
+distfiles="${homepage}/archive/v${version}.tar.gz"
```

Comment:
prefer using the PYPI_SITE

${PYPI_SITE}/M/MechanicalSoup/MechanicalSoup-${version}.tar.gz